script.js

```diff
@@ -0,0 +1,17 @@
+var mysql = require('mysql');
+
+var con = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "Shaigal5%",

```

Comment:
storing passwords in the repo is stronglyy discouraged.
In general, it is a good idea to not store the connection details in the source code, and this should be read from a configuration file.

Reply:
ok sim, how do you suggest to do it?
I'm not familliar with the process

Reply:
First of all, I suggest not connecting to the database directly from the client-side javascript.
As part of your ORM, you should have a place to configure the credentials.

Reply:
Ok, I'd like to schedule with you and talk about how to do it best for the project
Let's coordinate in slack :)